src/Tizen.NUI/src/public/GlWindow.cs

```diff
@@ -0,0 +1,535 @@
+            if (NDalicPINVOKE.SWIGPendingException.Pending) throw NDalicPINVOKE.SWIGPendingException.Retrieve();
+        }
+
+        /// <summary>
+        /// Allows at least one more render, even when paused.
+        /// The window should be shown, not minimised.
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public void RenderOnce()
+        {
+            Interop.GlWindow.GlWindow_RenderOnce(swigCPtr);
+            if (NDalicPINVOKE.SWIGPendingException.Pending) throw NDalicPINVOKE.SWIGPendingException.Retrieve();
+        }
+
+        internal void SetWindowSize(Size2D size)
+        {
+            var val = new Rectangle(Interop.GlWindow.GlWindow_GetPositionSize(swigCPtr), true);
+            Rectangle ret = new Rectangle(val.X, val.Y, size.Width, size.Height );
+
+            Interop.GlWindow.GlWindow_SetPositionSize(swigCPtr, Rectangle.getCPtr(ret));
+
+            if (NDalicPINVOKE.SWIGPendingException.Pending) throw NDalicPINVOKE.SWIGPendingException.Retrieve();
+        }
+
+        internal Size2D GetWindowSize()
+        {
+            var val = new Rectangle(Interop.GlWindow.GlWindow_GetPositionSize(swigCPtr), true);
+
+            Size2D ret = new Size2D(val.Width, val.Height);
+
+            if (NDalicPINVOKE.SWIGPendingException.Pending) throw NDalicPINVOKE.SWIGPendingException.Retrieve();
+            return ret;
+        }
+
+        /// <summary>
+        /// Type of callback to initialize native GL code.
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public delegate void GlInitializeType();
+
+        GlInitializeType init1;
+        HandleRef init2;
+
+        /// <summary>
+        /// Type of callback to render to frame to use native GL code.
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public delegate void GlRenderFrameType();
+
+        GlRenderFrameType render1;
+        HandleRef render2;
+
+
+        /// <summary>
+        /// Type of callback to cleanup native GL resource.
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public delegate void GlTerminateType();
+
+        GlTerminateType terminate1;
+        HandleRef terminate2;
+
+        /// <summary>
+        /// Registers a GL callback function for application.
+        /// </summary>
+        /// <param name="glInit">The callback function for application initialize</param>
+        /// <param name="glRenderFrame">The callback function to render to the frame</param>
+        /// <param name="glTerminate">The callback function to clean-up application GL resource</param>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public void RegisterGlCallback( GlInitializeType glInit, GlRenderFrameType glRenderFrame, GlTerminateType glTerminate )
+        {
+            init1 = glInit;
+            init2 = new HandleRef(this, Marshal.GetFunctionPointerForDelegate<Delegate>(init1));
```

Comment:
could you do naming the values as meaningful one?
also could you check this link?

https://docs.microsoft.com/ko-kr/dotnet/api/system.gc.keepalive?view=netcore-3.1

if these variables are for blocking garbage collected, then you can use KeepAlive maybe...